packages/stack/src/views/Header/HeaderTitle.tsx

```diff
@@ -1,12 +1,12 @@
 import * as React from 'react';
-import { Text, StyleSheet, Platform, TextProps } from 'react-native';
+import { Animated, StyleSheet, Platform, TextProps } from 'react-native';
 
 type Props = TextProps & {
   children?: string;
 };
 
 export default function HeaderTitle({ style, ...rest }: Props) {
-  return <Text {...rest} style={[styles.title, style]} />;
+  return <Animated.Text {...rest} style={[styles.title, style]} />;
```

Comment:
We're using Reanimated in stack. So we should change this to import `Animated` from `react-native-reanimated` instead.

Reply:
done 👍